test/SadTest.cc

```diff
@@ -587,8 +585,8 @@ class SadLoopTest : public ::testing::WithParamInterface<SadLoopTestParam>,
                       std::get<1>(TEST_GET_PARAM(1)), TEST_GET_PARAM(0),
                       std::get<0>(TEST_GET_PARAM(2)),
                       std::get<1>(TEST_GET_PARAM(2))) {
-        src_stride_ = width_ * 2;
-        ref1_stride_ = ref2_stride_ = 128;
+        src_stride_ = 256;
+        ref1_stride_ = ref2_stride_ = 720;
```

Comment:
define a macro to represent the max ref width and height, and set ref1_stride equal to the macro.